guides/06_FAQs.md

```diff
@@ -0,0 +1,239 @@
+
+## Demos
+
+### I want to build a React application that uses the Amazon Chime SDK. Do you have a sample application that I can reference?
+
+Amazon Chime has published a [blog post](https://aws.amazon.com/blogs/business-productivity/building-a-virtual-classroom-application-using-the-amazon-chime-sdk/) and [demo](https://github.com/aws-samples/amazon-chime-sdk-classroom-demo) for a building a virtual classroom using the Amazon Chime JS SDK.
+
+### I want to build a React Native application that uses the Amazon Chime SDK. Do you have a sample application I can reference?
+
+Amazon Chime SDK has published a react native demo that can be accessed [here](https://github.com/aws-samples/amazon-chime-react-native-demo).
+
+### Do I get billed for using the demos?
+
+Your account will get billed for attendee minutes and for any other AWS resources used by the demo. Please use [AWS Cost Explorer](https://aws.amazon.com/aws-cost-management/aws-cost-explorer/) for details on cost breakdown.
+
+### Does Amazon Chime SDK support recording functionality?
+
+No. Currently the Amazon Chime SDK does not support meeting recording out of the box. However, we have published a [blog post](https://aws.amazon.com/blogs/business-productivity/how-to-enable-client-side-recording-using-the-amazon-chime-sdk/) and a [demo](https://github.com/aws-samples/amazon-chime-sdk-recording-demo) demonstrating how to enable client-side recording using the Amazon Chime SDK.
+
+## Debugging
+
+### How can I get Amazon Chime SDK logs for debugging?
+
+Applications can get logs from Chime SDK by passing instances of Logger when instantiating [the MeetingSession object](https://github.com/aws/amazon-chime-sdk-js/blob/master/src/meetingsession/MeetingSession.ts#L14). Amazon Chime SDK has some default implementations of logger that your application can use, such as [ConsoleLogger](https://github.com/aws/amazon-chime-sdk-js/blob/8e5802c04031ac62a2c5c5fe75af5bb4ffbf12d0/src/logger/ConsoleLogger.ts) which logs into the browser console, [MeetingSessionPOSTLogger](https://github.com/aws/amazon-chime-sdk-js/blob/8e5802c04031ac62a2c5c5fe75af5bb4ffbf12d0/src/logger/MeetingSessionPOSTLogger.ts) which logs in [Amazon CloudWatch](https://aws.amazon.com/cloudwatch/) and [MultiLogger](https://github.com/aws/amazon-chime-sdk-js/blob/8e5802c04031ac62a2c5c5fe75af5bb4ffbf12d0/src/logger/MultiLogger.ts) which logs in multiple destinations.
+
+On mobile phones such as iPhone you can also do some advance debugging by connecting the phone to a Mac such as highlighted in this [article](https://developer.apple.com/library/archive/documentation/AppleApplications/Conceptual/Safari_Developer_Guide/GettingStarted/GettingStarted.html).
```

Comment:
Sorry, I gave you this link. This Safari doc is deprecated. I can't find an offical doc on how to debug the mobile Safari using the desktop. You can remove this sentence for now and re-add it with examples.